CMakeLists.txt

```
@@ -221,6 +221,23 @@ if(USE_CUDA)
   add_subdirectory(${PROJECT_SOURCE_DIR}/gputreeshap)
 
   find_package(CUDAToolkit REQUIRED)
+  find_package(CCCL CONFIG)
+  if(NOT CCCL_FOUND)
+    message(STATUS "Standalone CCCL not found. Attempting to use CCCL from CUDA Toolkit...")
+    find_package(CCCL CONFIG
+      HINTS ${CUDAToolkit_LIBRARY_DIR}/cmake)
+    if(NOT CCCL_FOUND)
+      message(STATUS "Could not locate CCCL from CUDA Toolkit. Using Thrust and CUB from CUDA Toolkit...")
+      find_package(Thrust CONFIG REQUIRED
+        HINTS ${CUDAToolkit_LIBRARY_DIR}/cmake)
```

Think we also need to call [`thrust_create_target`]( https://github.com/NVIDIA/cccl/blob/1b16af7ccc2f3aeb4c94a743c951bae3e0586052/thrust/thrust/cmake/README.md?plain=1#L14-L23 )

```suggestion
      find_package(Thrust CONFIG REQUIRED
        HINTS ${CUDAToolkit_LIBRARY_DIR}/cmake)
      thrust_create_target(Thrust)
```